app/controllers/api/v1/application_users_controller.rb

```diff
@@ -92,6 +92,33 @@ def index
   end
 
   ###############################################################
+  # username
+  ###############################################################
+
+  api :GET, '/application_users/username/:username',
+      'Gets a single User with the specified username.'
+  description <<-EOS
+      All users of OpenStax have an associated User object.  This endpoint
```

Comment:
This description should be changed to reflect that this endpoint deals with ApplicationUsers, not Users (reading it it sounds like it is searching for and returning Users).  Fine to say that the response will include information in the User record if helpful to a developer.